recipe/build.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# We use a repackaged cmake from elsewhere to break a build cycle.
+export PATH=${PREFIX}/cmake-bin/bin:${PATH}
```

Comment:
@nehaljwani I got this from your recipe and now ti works, but `ctest` fails.
I see that you are not running them, any advice?

Reply:
For posterity sake, the build that failed was: https://circleci.com/gh/conda-forge/libssh2-feedstock/24
I'll look into why the ctest fails inside docker container this weekend.

I didn't modify the libssh2-feedstock in AnacondaRecipes. That was @mingwandroid 's work.

Reply:
Thanks for the info @nehaljwani!